### DISCUSSION TITLE
Keine Power anzeige bei RRDstats

### DISCUSSION THREAD

**Discussion by freetz-ng-mod** [Freetz-NG/freetz-ng#510] [Off-topic]
Hi @fda77 auf den folgenden Geräten wird mir keine Power anzeige gezeigt unter RRDstats, 1200AX und 7590AX. Auf den anderen Geräten geht die Anzeige. Ein Tipp was ich checken kann, woran es liegt

**Comment by fda77** [comment]
Also wird "0" oder "NaN" angezeigt?

**Comment by freetz-ng-mod** [comment]
1200AX
![1200ax](https://user-images.githubusercontent.com/79326105/161435062-b87ba66f-66d5-4de1-9dc0-a86dbd3a1c19.png)
7590ax
![7590ax](https://user-images.githubusercontent.com/79326105/161435073-2d0a9862-409f-4c6f-a344-d7dc7901c667.png)

**Comment by fda77** [comment]
Vielleicht hast du die falschen konfiguriert?

**Comment by freetz-ng-mod** [comment]
Vielleicht ist auch auf allen Systemen die gleiche. Und da lauft es 
5530 6000 7490 7530AX

**Comment by fda77** [comment]
Ne, jedes System hat andere die man auslesen kann. Das wird beim konfigurieren auch angezeigt. Deshalb frag ich auch!

**Comment by fda77** [comment]
Beim 6000 wird zb usb zu 100% nicht funktionieren (hats nicht)

**Comment by freetz-ng-mod** [comment]
7590AX
Verfügabr: sum=39 system=69 dsp=100 wlan=100 eth=0 ab=0 battcharge=0 dect=22 usbhost=0 lte=0
sum system dsp wlan usbhost ab eth dect
1200AX
Verfügabr: sum=87 system=71 dsp=0 wlan=100 eth=2 ab=0 battcharge=0 dect=0 usbhost=0 lte=0
sum system wlan eth

Und das ist auch drin. Glaub mir einfach es wird nicht angezeigt

**Comment by fda77** [comment]
Die mit "=0" würde ich nicht konfigurieren, da diese meist erst höhere Werte haben wenn die Funktion im AVM-Webif aktiviert wird.
Und da die anderen plausible Werte anzeigen sollten die die vom Script ausgelesen werden können.
Kann ja selbst über SSH mit "ctlmgr_ctl"  ausprboieren:
https://github.com/Freetz-NG/freetz-ng/blob/master/make/rrdstats/files/root/usr/bin/rrdstats#L106=

**Reply by fda77** [reply]
$sourceitem sind dann die Namen wie "sum"

**Comment by freetz-ng-mod** [comment]
Schaue ich mir später an und Danke
